recipes/conda-forge-pinning/conda_build_config.yaml

```diff
@@ -0,0 +1,215 @@
+blas_impl:
+  - mkl                        # [x86 or x86_64]
+  - openblas
+arpack:
+  - 3.5
+boost:
+  - 1.65
+bzip2:
+  - 1.0
+cairo:
+  - 1.14
+# This differs from target_platform in that it determines what subdir the compiler
+#    will target, not what subdir the compiler package will be itself.
+#    For example, we need a win-64 vs2008_win-32 package, so that we compile win-32
+#    code on win-64 miniconda.
+cross_compiler_target_platform:  # [win]
+  - win-32                     # [win]
+  - win-64                     # [win]
+curl:
+  - 7.54
+c_compiler:                    # [win]
+  - vs2008                     # [win]
+  - vs2015                     # [win]
+  - vs2015                     # [win]
+cxx_compiler:                  # [win]
+  - vs2008                     # [win]
+  - vs2015                     # [win]
+  - vs2015                     # [win]
+fortran_compiler:              # [win]
+  - intel-fortran              # [win]
+fortran_compiler_version:      # [win]
+  - 13.1.6                     # [win]
+  - 16.0.4                     # [win]
+  - 16.0.4                     # [win]
+CONDA_BUILD_SYSROOT:           # [osx]
+  - /opt/MacOSX10.9.sdk        # [osx]
+VERBOSE_AT:
+  - V=1
+VERBOSE_CM:
+  - VERBOSE=1
+dbus:
+  - 1
+expat:
+  - 2.2                # [not ppc64le]
+ffmpeg:
+  - 3.2.3
+flann:
+  - 1.9.1
+fontconfig:
+  - 2.12
+freetype:
+  - 2.8                # [not ppc64le]
+gstreamer:
+  - 1.12
+gst_plugins_base:
+  - 1.12
+geos:
+  - 3.6.2
+giflib:
+  - 5.1
+glib:
+  - 2.53
+glpk:
+  - 4.62
+gmp:
+  - 6.1
+harfbuzz:
+  - 1.7
+hdf4:
+  - 4.2
+hdf5:
+  - 1.10               # [not ppc64le]
+icu:
+  - 58                 # [not ppc64le]
+ipython:
+  - 5.4                # [not ppc64le]
+  - 5.3                    # [ppc64le]
+  - 6.1
+  - 6.1
```

Comment:
Why does `ipython` need pinning or am I misunderstanding what this does?

Reply:
Because IPython drops support for python 2.7 with the 6.0 rev.  Pay special attention to zip_keys.  It's associating these ipython versions with python versions.

Reply:
> Because IPython drops support for python 2.7 with the 6.0 rev. 

I don't get why we need pinning even with the above reason. If a package is built with ipython 6.1, it won't work with 6.2?

Reply:
I believe it is meant to enforce or limit building of things that use ipython.  I'm not sure how necessary it is - ``ipython >=4`` would probably be functionally equivalent, since the ipython 6.x packages are not available for python 2.

@mingwandroid I think you added that in.  Any comment on what this is for?

Reply:
It is to align the zip_keys. I don't know if >=4 will work, I don't know much about ipython.

Reply:
AFAIUI, you cannot use >=4 in this file, it specifies a list of version numbers to build against.

Reply:
Opened issue ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/1 ) on the feedstock so that we can finish the discussion over there and figure out what to do (if anything).